script.js

```diff
@@ -0,0 +1,21 @@
+function sendEmail(){
+    var name=document.getElementById("Name").value;
+    var email=document.getElementById("Email").value;
+    var message=document.getElementById("message").value;
+
+    if(name==""){
+       alert("Please enter Your Names");
+       return false;
+    }
+    else if(email==""){
+        alert("Please enter Your user Email");
```

Comment:
What if someone puts an email without an @ or .com?! or put anything.. i think your email input will accpet them which isn't a good approach. i urge you to work on this!